tests/unit/test_avatar_interaction_runtime.py

```
@@ -0,0 +1,315 @@
+    fist_instruction = _build_avatar_interaction_instruction(manager, {
+        "tool_id": "fist",
+        "action_id": "poke",
+        "intensity": "normal",
+        "reward_drop": False,
+        "easter_egg": False,
+        "touch_zone": "head",
+        "text_context": "",
+    })
+
+    assert "接触位置" not in lollipop_instruction
+    assert "接触位置：头顶" in fist_instruction
```

Ruff (0.15.9)

[warning] String contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF001)